android/src/ti/map/AnnotationProxy.java

```diff
@@ -107,6 +104,12 @@ public void release()
 		super.release();
 	}
 
+	public void releaseMarker() {
+		if (marker != null) {
```

Comment:
We should probably release any custom views too:
```Java
if (hasProperty(MapModule.PROPERTY_CUSTOM_VIEW)) {
 	TiViewProxy customView = (TiViewProxy) getProperty(MapModule.PROPERTY_CUSTOM_VIEW);
 	customView.releaseView();
 }
```

Reply:
I gave that a shot. We end up being unable to reuse the custom views - the same scenario we had with the annotations.
I tested with the profiler and it seems the references to the custom view are release when a proxy "out of context" is garbage collected and we do not have the custom views leaking. So I think we are fine with releasing the custom view when the AnnotationProxy is released.